tests/text/generate/test_sequence.py

```diff
@@ -67,38 +67,6 @@ def __hash__(self):
         return id(self)
 
 
-def test_vectorized_random_choice():
```

Comment:
Shouldn't this function still be tested?

Reply:
It was moved.